src/store/Workspaces/index.ts

```diff
@@ -369,12 +377,17 @@ type StatePartial = {
   [key in keyof State]: State[key];
 };
 
+const maxLogLength = 200;
+
 const mapMerge = (originMap: Map<string, string[]>, additionalMap: Map<string, string[]>): Map<string, string[]> => {
   if (!originMap.size) {
     return additionalMap;
   }
   const res = new Map<string, string[]>();
   originMap.forEach((val: string[], key: string) => {
+    if (val.length > maxLogLength) {
+      val.splice(0, maxLogLength);
```

Comment:
Why do you need to trim long lines? Maybe, splitting a long line in a few short lines would be better? or what about set `overflow: hidden` to the container where logs are displayed?

Reply:
We discussed this offline and decided to implement it the way that allows users to download full logs.